pymc3/distributions/.dir-locals.el

```diff
@@ -0,0 +1,5 @@
+;;; Directory Local Variables
```

Comment:
Accidental commit.

Reply:
Thanks for catching that.  Fixed in rebase, so it's removed from the history (wait for force push).